assets/_scss/elements/_list.scss

```diff
@@ -2,24 +2,53 @@ ul, ol {
   margin: {
     top: 2em;
     bottom: 2em;
-  padding-left: 2em;
   }
 
   li {
     line-height: $base-line-height;
     margin: {
-      top: 1em;
-      bottom: 1em;
+      top: .75em;
+      bottom: .75em;
     }
   }
 }
 
-ul {
-  list-style-type: square;
+ul, ol {
+  list-style: none;
+  padding-left: 0;
+  display: table;
+
+  li:after {
+    content: "";
+    display: block;
+    margin-bottom: 0.5em;
+  }
 }
 
-ol {
-  list-style-type: decimal;
+ul li {
+  display: table-row;
```

Comment:
@msecret in order to get multi-digit list alignment to flush left. See this issue: https://github.com/18F/govt-wide-patternlibrary/issues/42#issuecomment-118968237